### DISCUSSION TITLE
Account login problem

### DISCUSSION THREAD

**Discussion by mit0011** [bitshares/bitshares-core#2817] [Q&A]
Hello. I'll start from the beginning
The situation is as follows. I created an account on wallet.bitshares.org 1.11.24, but did not make a backup key. When I decided to log into the account on 5.11.24, the password didn't work. I keep all my passwords in Keepassxc though. I emailed an admin from the xbts group on tg. He said that if there was no backup key, no one would help, and suggested creating a new account.

I created a new account on 6.11.24, made a backup key. Entered password at registration with ctrl c, ctrl v. via buffer. Decided to check, restarted the browser, when I tried to log into the account the password was incorrect. Tried to restore with the backup key, also did not help. 
Please help me to restore at least the last account. Please advise how to contact you? I provide: login, password and key, as well as I can provide video materials, dates when the registration was made.

**Comment by abitmore** [comment]
If you created your account in the "cloud wallet" mode which is the default option, you need to store your account name AND your password safely, no other backup methods. And then you should be able to login with the account name and the password anywhere and at any time, in the same "cloud wallet" mode. Ctrl C / Ctrl V should be OK. If it didn't work, please check if you have made any typo.

If you used the "local wallet" or "key file" mode which is an advanced option, you should store the backup (bin file) safely so that you can restore it later. Note that the password in this case is to encrypt the bin file, and you need to store it safely too.

Don't mix the two modes if you don't know how it works. Nobody can restore your account except yourself.

**Comment by abitmore** [comment]
BTW on TG you can ask in the [@btsWalletHelp](https://t.me/btsWalletHelp) group or reach me at [@abitmore](https://t.me/abitmore).

**Reply by mit0011** [reply]
I emailed you at TG and sent you a video of logging into the account, using a .bin file

**Comment by abitmore** [comment]
It is a blockchain. You are responsible for your own private keys and passwords.
